styleguide/derek/_partials/solutions/leadSplit.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Lead pattern -->
+<!--
+.lead class naming not used in alignment with typical pattern structure
+instead here .lead-wrapper is used to avoid having to override bootstrap base class
+ -->
+<div class="lead-wrapper">
+  <div class="lead-headerWrapper">
+    <h1 class="lead-header left">Title goes here</h1>
```

Comment:
We use this with callouts, videos and pdf downloads. Confirming with Mike as we speak. I'd rather not create an all new class since lead already does this in www right?